specs/electra/beacon-chain.md

```diff
@@ -258,6 +259,17 @@ class PendingConsolidation(Container):
     target_index: ValidatorIndex
 ```
 
+
+#### `SingleAttestation`
+
+```python
+class SingleAttestation(Container):
+    committee_index: CommitteeIndex
+    attester_index: ValidatorIndex
+    data: AttestationData
+    signature: BLSSignature
```

Comment:
Why is `committee_index` here? Doesn't `data.index` represent the committee index?

https://github.com/ethereum/consensus-specs/blob/ca04b1e94de25e224308640877ef96b98ba9b119/specs/phase0/beacon-chain.md?plain=1#L373-L380

Reply:
It does not with the change proposed by the [EIP-7549](https://github.com/ethereum/consensus-specs/pull/3559)

Reply:
Ah duh. Sorry & thanks!